.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: node_js
+node_js: "13"
+
+services:
+  docker
+
+jobs:
```

Comment:
Travis runs unit tests, e2e API tests and API analysis in three jobs not unlike concourse but running off docker-compose